proto/rill/local/v1/api.proto

```diff
@@ -42,3 +51,56 @@ message GetVersionResponse {
   string current = 1;
   string latest = 2;
 }
+
+message DeployValidationRequest {}
+
+message DeployValidationResponse {
+  bool is_authenticated = 1; // if true below fields are relevant after login
+  bool is_github_connected = 2; // if true below fields are relevant after git install
+  string git_grant_access_url = 3; // redirect to this if is_github_connected or is_repo_access_granted is false
+  string git_user_name = 4;
+  bool is_user_app_installed = 5; // if rill git app is installed on user account
+  string user_app_permission = 6;
```

Comment:
1. Same question as in the admin protos
2. Since it's Github-related, put a ` github_` prefix in the name
3. Since it seems related to `organizations_with_app` (both dealing in Github access permissions), can we move that up next to this and have similar names between the two?